modules/file-rotation/file-rotation.c

```diff
@@ -22,29 +22,44 @@
  */
 
 #include "file-rotation.h"
+#include "modules/affile/file-signals.h"
 #include "driver.h"
 
-struct _FileRotationPlugin
+static void
+_slot_file_rotation(FileRotationPlugin *self, FileFlushSignalData *data)
 {
-  LogDriverPlugin super;
-  gsize size;
-};
+  msg_trace("File rotation signal received", evt_tag_str("filename", self->filename->str));
```

Comment:
Does not this crash?
Currently, self->filename is probably a NULL pointer.

Reply:
Updated 👍